history.adoc

```diff
@@ -7,6 +7,7 @@
 
 === Working version (most recent first)
 
+* {issues}141[Issue #141]: Clarification that text-valued variables and attributes can be Unicode vlen strings or UTF-8 char arrays.
```

Comment:
Maybe: 

Clarification that text-valued variables and attributes can be vlen strings or char arrays, encoded at UTF-8.

though this is the history, so precision isn't critical.

Reply:
Looks great, thanks for all the work!

Reply:
Changed to "Clarification that text may be stored in variables and attributes as either vlen strings or char arrays, and must be represented in Unicode Normalization Form C and encoded according to UTF-8."